.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
+}
```

Comment:
I think the .vscode folder doesn't need to be versioned either

Reply:
deleted the file @tawfeeka